### DISCUSSION TITLE
Is Galaxy Brain Badge still achievable?

### DISCUSSION THREAD

**Discussion by matheusaudibert** [community/community#134097] [Discussions] [answered]
### Select Topic Area

Question

### Body

I was trying to get all the GitHub badges and the last one that I need is the Galaxy Brain, I alredy answered more than 2 discussion and still dont got it, anyboy know if the bagde is close or sm like this?
![Uploading imagem_2024-07-30_141917968.png…]()

**Comment by davevad93** [comment, marked as answer]
Hi @matheusaudibert ,

Yes, the `Galaxy Brain` badge can still be obtained, but not in the Community Discussions, for more info about the reasons see [here](https://github.com/orgs/community/discussions/106536). 
You can still get it by answering discussions outside the Community, for example public repos with open discussions.

**Reply by LiteBrite82** [reply]
@matheusaudibert 
But also, be like @davevad93, Jump in and get involved with the community; lets grow together!

**Comment by RidmaShehan** [comment]
Hi 👋
The Galaxy Brain badge is not instant, even if you’ve already answered more than two discussions.
A few important things to know:
The badge is awarded when your answers are marked as “Accepted” (not just posted).
It usually requires multiple accepted answers (often 2 or more).
Badge updates are not real-time — it can take days or even weeks for GitHub to process and show the badge.
Sometimes GitHub rolls out badges gradually, so there can be delays even if you qualify.
So if your answers have been accepted, you’re likely eligible — you just need to wait a bit. If they haven’t been accepted yet, that’s probably why the badge hasn’t appeared.
Hope that helps! 😊
